src/components/BookForm.jsx

```diff
@@ -26,28 +26,61 @@ const BookForm = () => {
   };
 
   return (
-    <div>
-      <h2>Add Book</h2>
-      <form onSubmit={handleSubmit}>
+    <div className="mx-16 mb-16">
+      <h2 className="text-xl font-bold mb-3">ADD NEW BOOK</h2>
+      <form onSubmit={handleSubmit} className="w-full flex justify-between items-center">
         <input
           type="text"
           value={title}
           onChange={(e) => setTitle(e.target.value)}
-          placeholder="Enter book title"
+          placeholder="Book title"
+          className="w-1/3 h-12 mr-4 pl-3 pr-14 rounded-md border border-gray-300 bg-white"
         />
         <input
           type="text"
           value={author}
           onChange={(e) => setAuthor(e.target.value)}
-          placeholder="Enter book author"
+          placeholder="Author"
+          className="w-1/3 h-12 mr-4 pl-3 pr-14 rounded-md border border-gray-300 bg-white"
```

Comment:
- [ ] The point of this milestone is to follow the given [zeplin](https://zeplin.io/) design, this includes the font-family used in the original design. Kindly apply this to your project as well. Please note that there are two different fonts applied throughout this project, `RobotoSlab` and `Montserrat`

Reply:
done